lib/graph.py

```diff
@@ -66,18 +75,14 @@ def __str__(self):
                 for key, value in self.style.items()]))
 
 
-
 class CommandNode(Node):
-    def __init__(self, command, truncate=30, **style):
-        self.command = command
+    def __init__(self, description, command, truncate=40, **style):
```

Comment:
Suggest changing the `truncate` argument to `line_width`, since we're no longer truncating.